rpcserver.go

```diff
@@ -7072,19 +7088,27 @@ func (r *rpcServer) ForwardingHistory(ctx context.Context,
 	for i, event := range timeSlice.ForwardingEvents {
 		amtInMsat := event.AmtIn
 		amtOutMsat := event.AmtOut
-		feeMsat := event.AmtIn - event.AmtOut
+		feeMsat := int64(event.AmtIn) - int64(event.AmtOut)
+
+		// Get unsigned representation of the fee. Report zero if
```

Comment:
Q: Why do we not want to show negative fees in case we did overpay in forwards rather than earning fees? Is it because of backwards compatibility ?

Reply:
Yes, backwards compatibility indeed. Wanted to keep the proto datatype unchanged.